platform/FilePath.cpp

```diff
@@ -18,9 +18,10 @@
 namespace mbed {
 
 FilePath::FilePath(const char* file_path) : file_name(NULL), fb(NULL) {
-    if ((file_path[0] != '/') || (file_path[1] == 0)) return;
+    // skip slashes
+    file_path += strspn(file_path, "/");
```

Comment:
That's an amusing way to catch a slew of slashes.

Reply:
What's more amusing is the rest of the function could be replaced by:
``` cpp
len = strcspn(file_path, "/");
```